### DISCUSSION TITLE
Community Discord

### DISCUSSION THREAD

**Discussion by evoactivity** [TheWidlarzGroup/react-native-video#3659] [General]
Hello all 👋 

I've setup a discord for everyone to talk and discuss React Native Video.

I'm hoping we can build a community of like-minded developers who are willing to help each other out when using this library.

https://discord.gg/j3k5UTRg

As it's a new server I want to keep things limited for now and reduce chances of spam so this invite will expire in 7 days. If you try to enter and it has expired please post a reply here and I'll create a new invite link to share.

Happy to take suggestions for server improvements and new channels.

**Comment by KrzysztofMoch** [comment]
Hey there's already a discord server for `react-native-video` (you can see it during bug report) 
here's the link: https://discord.gg/WXuM4Tgb9X. 
If you are interested in administrating the section about react-native-video let's catch up 😄

**Reply by evoactivity** [reply]
Oh, well then, this was confusing lol
https://github.com/react-native-video/react-native-video/issues/2876#issuecomment-1824956934 I messaged @freeboub on slack and on that issue to get confirmation but got no replies. Decided to just share the discord 😅

**Reply by KrzysztofMoch** [reply]
Sorry for confusing 🙏
But if you are interested in making such server you can cooperate with us
BTW: your server looks very nice!
